index.bs

```diff
@@ -1058,17 +1058,18 @@ noticeable asymmetry between the two branches, and limits the possible <a>chunks
   1. Let _pullAlgorithm_ be the following steps:
     1. Return the result of <a>transforming</a> ! ReadableStreamDefaultReaderRead(_reader_) with a fulfillment handler
        which takes the argument _result_ and performs the following steps:
+      1. If _closed_ is *true*, return.
       1. Assert: Type(_result_) is Object.
-      1. Let _value_ be ? Get(_result_, `"value"`).
-      1. Let _done_ be ? Get(_result_, `"done"`).
+      1. Let _done_ be ! Get(_result_, `"done"`).
```

Comment:
Interesting! If I understand correctly, this works because we know `result` has a `null` prototype, so we always go through [`Get`](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-get-o-p) -> [`[[Get]]`](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-ordinary-object-internal-methods-and-internal-slots-get-p-receiver) -> [`OrdinaryGet`](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-ordinaryget) -> [`[[GetOwnProperty]]`](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-ordinary-object-internal-methods-and-internal-slots-getownproperty-p) -> [`OrdinaryGetOwnProperty`](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-ordinarygetownproperty), which cannot throw.

For the same reason, I think we can also change `? Get` to `! Get` in [`ReadableStreamAsyncIteratorPrototype.next()`](https://streams.spec.whatwg.org/commit-snapshots/a441f59db468c3a40515cc2e7590abe381632f8d/#rs-asynciterator-prototype-next)?

Reply:
> For the same reason, I think we can also change `? Get` to `! Get` in [`ReadableStreamAsyncIteratorPrototype.next()`](https://streams.spec.whatwg.org/commit-snapshots/a441f59db468c3a40515cc2e7590abe381632f8d/#rs-asynciterator-prototype-next)?

Yes. Feel free to send a PR! :smile:

Reply:
➡️ #995